README.md

```diff
@@ -42,6 +44,25 @@ In Mac OS X, PFNET can be easily installed using [Homebrew](http://brew.sh):
 Since Homebrew does currently not support external dependencies, PFNET cannot be built with raw parsing capabilities with this method.
 
 Homebrew will install graphviz as a dependency unless ``--without-graphviz`` is specified.
+
+If building from source directly. The Build Instructions for Linux should suffice but substitute ``$DYLD_FALLBACK_LIBRARY_PATH`` for ``$LD_LIBRARY_PATH``.
+
+## Build Documentation (Linux) ##
+
+Building the documentation for PFNET requires [Doxygen](http://www.stack.nl/~dimitri/doxygen/) (for C) and [Sphinx](http://www.sphinx-doc.org/en/stable/) (for Python). The environment variable ``PFNET_DOCS`` must be set to the location where the documentation will be moved to once it is built.

```

Comment:
Instructions for Python should be in the python docs. We should not mix Python-specific things with things for for the C library.